spec/user_spec.rb

```diff
@@ -0,0 +1,59 @@
+require_relative '../lib/person.rb'
+require_relative '../lib/user.rb'
+
+describe User do
+
+  context '#print_age_based_message' do
+    
+    let(:user) { User.new(age: 100) }
+    describe "is a century old" do
+      describe "is older than dirt" do
+        it "is true" do
```

Comment:
The descriptions are testing that `#print_age_based_message` will return `true` if they are 100 years old -- which is a bit confusing. If you change this `it` block to have a description of `"returns correct message"` then it makes more sense.